docs/speech-ui-api-docs.md

```diff
@@ -16,19 +16,10 @@ npm install
 npm start
 ```
 
-You should see a new icon in the toolbar. Click it and a small window will open. Use the keyboard shortcut for Chrome developer tools to open up the console. You'll need to type in the following command to enter the API key before using it:
-
-```javascript
-window.WITAI_API_SECRET = "Bearer XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
-```
-
-Ask for the API key or get a free one from wit.ai.
-
-
 # Flask API
 
 - Install Flask with the command: pip install flask
```

Comment:
Should we just use `pip3 install -r requirements.txt` instead? The new requirements file has Flask already...https://github.com/castorini/Anserini/blob/master/requirements.txt